Project.toml

```diff
@@ -1,17 +1,22 @@
 name = "DustExtinction"
 uuid = "fb44c06c-c62f-5397-83f5-69249e0a3c8e"
 license = "MIT"
-version = "0.4.0"
+version = "0.5.0"
 
 [deps]
 FITSIO = "525bcba6-941b-5504-bd06-fd0dc1a4d2eb"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
```

Comment:
Maybe just require next version of `UnitfulAstro` (which will require `Unitful` 0.17 after https://github.com/JuliaAstro/UnitfulAstro.jl/pull/13 is merged)?  We don't directly need `Unitful` here, right?

Reply:
Well, we do because we can’t access things like `ustrip` without ```using Unitful```. This means we need to explicitly add `Unitful` as a dep. Correct me if I’m wrong.

Reply:
You're right, it's just me making confusion between the two pull requests :sweat_smile: